nix/nixos/cardano-tracer-service.nix

```diff
@@ -0,0 +1,89 @@
+pkgs:
+let serviceConfigToJSON =
+      cfg:
+      {
+        networkMagic = 764824073; ## Mainnet
+        # loRequestNum = 100;
+        network =
+          if        cfg.acceptingSocket != null
+          then {
+            tag      = "AcceptAt";
+            contents = cfg.acceptingSocket;
+          } else if cfg.connectToSocket != null
+          then {
+            tag      = "ConnectTo";
+            contents = cfg.connectToSocket;
+          } else
+            throw "cardano-tracer-service:  either acceptingSocket or connectToSocket must be provided.";
+        logging = [{
+          inherit (cfg) logRoot;
+
+          logMode   = "FileMode";
+          logFormat = "ForMachine";
+        }];
+        rotation = {
+          rpFrequencySecs = 15;
+          rpKeepFilesNum  = 10;
+          rpLogLimitBytes = 500000000;
+          rpMaxAgeHours   = 24;
+        };
+
+        hasEKG = [
+          { epHost    = "127.0.0.1";
+            epPort    = 3100; ## supervisord.portShiftPrometheus
+          }
+          { epHost    = "127.0.0.1";
+            epPort    = 3101; ## supervisord.portShiftPrometheus
+          }
+        ];
+        ekgRequestFreq = 1;
+        hasPrometheus = {
+          epHost    = "127.0.0.1";
+          epPort    = 3200; ## supervisord.portShiftPrometheus
+        };
```

Comment:
After RTView will be merged in `master` (https://github.com/input-output-hk/cardano-node/pull/3852), `hasRTView` will be added here.